snapcraft/internal/remote_build/_launchpad.py

```diff
@@ -0,0 +1,233 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2019 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import os
+import shutil
+import time
+import urllib.request
+import urllib.error
+import urllib.parse
+
+from lazr import restfulclient
+from launchpadlib.launchpad import Launchpad
+from typing import Any, Dict, List
+from xdg import BaseDirectory
+from snapcraft.project import Project
+from .errors import (
```

Comment:
why not import "errors" instead? Optional, but it follows the pattern used in other packages we have

Reply:
I'm all for consistency, changed to import errors.